test/git_chain/command/delete_test.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+require "test_helper"
+
+module GitChain
+  module Commands
+    class DeleteTest < MiniTest::Test
+      include RepositoryTestHelper
+
+      def test_deleting_a_clean_chain
+        capture_io do
+          with_test_repository("a-b-chain") do
+            assert_equal(%w(master a b), Models::Chain.from_config("default").branch_names)
+
+            Delete.new.call
+
+            assert(Models::Chain.from_config("default").empty?)
```

Comment:
I'm not sure how to run these tests, but by following the pattern in other tests, I think these 2 should do what I want (verify that a chain is deleted, verify that delete is cancelled if a rebase is in progress).